docs/writing-tests/stories-in-unit-tests.md

```diff
@@ -2,17 +2,17 @@
 title: 'Unit tests'
 ---
 
-Teams test a variety of UI characteristics using different tools. Each tool requires you to replicate the same component state over and over. That’s a maintenance headache. Ideally, you’d set up your tests in the same way and reuse that across tools.
+Teams test a variety of UI characteristics using different tools. Each tool requires you to replicate the same component state over and over. That’s a maintenance headache. Ideally, you’d set up your tests similarly and reuse that across tools.
 
-Storybook enables you to isolate a component and capture its use cases in a `*.stories.js|ts` file. Stories are standard JavaScript modules cross-compatible with the whole JavaScript ecosystem.
+Storybook enables you to isolate a component and capture its use cases in a `*.stories.js|ts` file. Stories are standard JavaScript modules that are cross-compatible with the whole JavaScript ecosystem.
 
 Stories are a practical starting point for UI testing. Import stories into tools like [Jest](https://jestjs.io/), [Testing Library](https://testing-library.com/), [Vitest](https://vitest.dev/) and [Playwright](https://playwright.dev/), to save time and maintenance work.
 
 <IfRenderer renderer='vue'>
 
 ## Set up the testing addon
 
-Storybook's [`@storybook/testing-vue3`](https://storybook.js.org/addons/@storybook/testing-vue3/) addon is a powerful tool that simplifies the testing process by allowing you to reuse your stories inside alongside their associated mocks, dependencies, and context, saving time and ensuring consistency and accuracy in the testing process.
+By default, Storybook provides a set of APIs that enables you to run your stories as tests with the industry-standard testing tools, simplifying the testing process by allowing you to reuse your stories inside alongside their associated mocks, dependencies, and context. However, if you haven't upgraded to the latest stable Storybook release yet, you'll need to add the [`@storybook/testing-vue3`](https://storybook.js.org/addons/@storybook/testing-vue3/) addon to use these APIs.
```

Comment:
The Storybook 8.0 documentation shouldn't mention the @storybook/testing-vue3 package at all. This is part of the migration guide. We have a version switcher for the documentation if someone wants to check out how to do testing in Storybook prior to 8.

Reply:
I'll leave @jonniebigodes to this one! WDYT?

Reply:
I agree with @valentinpalkovic 's reasoning, as this is now the default for Storybook 8 onwards. We can remove this section and adjust the snippets by removing the reference to the addon to prevent any issues that may arise when using the addon with the upcoming version.

Reply:
Done!